### PR TITLE
Make the requirement on Python clear for GLib. Fix python shebang.

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/d522f9979a47e58d88191f8fc8f01c09 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-s1.int.private - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.14
See https://gist.github.com/7344efd5db666c237f6a09a003f0e71b for a full test report.

File: easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -14,10 +14,18 @@ sources = [SOURCELOWER_TAR_XZ]
 checksums = ['89f884f5d5c6126140ec868cef184c42ce72902c13cd08f36e660371779b5560']
 
 builddependencies = [
+    # Python is required for using GLib to build, at least when
```
Comment (boegel) [inline easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb]: `for building against GLib`?

File: easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -34,7 +42,7 @@ preconfigopts = "export PYTHON=python && "
 
 configopts = "--buildtype=release --default-library=both "
 
-postinstallcmds = ["sed -i -e 's|#!.*python|#!/usr/bin/env python|' %(installdir)s/bin/*"]
+postinstallcmds = ["sed -i -e 's|#!.*python[0-9.]*$|#!/usr/bin/env python|' %(installdir)s/bin/*"]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb]: Another good use case for https://github.com/easybuilders/easybuild-framework/issues/2869

Doesn't need to block this PR, but we should change this easyconfig once we have global support for `fix_python_shebang_for`...

File: easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/g/GLib/GLib-2.60.1-GCCcore-8.2.0.eb]: Text adjusted.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a51d2824e822509c587469807be58f8d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2557.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/82c3d254e27ae4c68eeb582745fc70fc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3122.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/f3d4f854f5e5b0dd60eb6a5154360563 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**